.env.example

```
@@ -0,0 +1,22 @@
+# syslog-mcp environment variables
+# Copy to .env and adjust for your environment.
+# All vars use SYSLOG_MCP__ prefix with __ for nesting.
+
+# Syslog listener bind addresses
+SYSLOG_MCP__SYSLOG__UDP_BIND=0.0.0.0:1514
+SYSLOG_MCP__SYSLOG__TCP_BIND=0.0.0.0:1514
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The SYSLOG_MCP__SYSLOG__TCP_BIND key should go before the SYSLOG_MCP__SYSLOG__UDP_BIND key

(UnorderedKey)